python/src/lib.rs

```diff
@@ -121,6 +121,43 @@ impl RawDeltaTable {
         })
     }
 
+    #[classmethod]
+    #[pyo3(signature = (table_uri, version = None, storage_options = None, without_files = false))]
+    fn load_lazy(
```

Comment:
What is the difference between `load_lazy` and `new` above?

Reply:
`load_lazy` uses `builder.build()`, while `new` uses `builder.load()`.  `builder.load()` build the DeltaTable and load its state. `builder.build()` only build DeltaTable. See details here: https://github.com/delta-io/delta-rs/blob/main/crates/deltalake-core/src/table/builder.rs#L269-L293

Reply:
Not sure if we need to include all the options in `load_lazy`